src/main/java/org/terasology/fluid/system/FluidAuthoritySystem.java

```diff
@@ -52,6 +57,34 @@
     private FluidRegistry fluidRegistry;
     @In
     private InventoryManager inventoryManager;
+    
+    @In
+    private BlockManager blockManager;
+    private Block air;
+    
+    @In
+    private ExtraBlockDataManager extraDataManager;
+    private int flowIx;
```

Comment:
Is there any specific meaning to "Ix" here? Might be some convention I'm unfamiliar with

Reply:
I asked the same in the other PR 🙈 : It stands for `index`. As @Cervator now is the second one having trouble with this abbreviation I'd like to ask you to change this to `flowIndex` for more clarity (the additional characters won't hurt us 🙃 ):
```suggestion
    private int flowIndex;
```

Reply:
Okay, will do. How do you make suggested changes like that on Github?

Reply:
Just a cool new feature during a review - there's a button for "Insert Suggestion" 👍 

![image](https://user-images.githubusercontent.com/1063833/88691303-50706f00-d0c2-11ea-8fdf-8cf146b8c855.png)